Settings/CommerceSettings.cs

```diff
@@ -9,5 +9,7 @@ public class CommerceSettings
         /// The default currency ISO code
         /// </summary>
         public string DefaultCurrency { get; set; }
+
+        public string CurrentDisplayCurrency { get; set; }
```

Comment:
I think too many of these features are being added to the base feature of the module, adding complexity that not everybody needs. While the base abstractions for all extensibility points should be there, it seems to me like actual multi-currency shops will be the minority, and as such should be isolated in separate features that have to be deliberately activated.

Reply:
I agree that this should be a feature. For now I think it's good to have it always available during early development to make it easier to find issues.

Reply:
I think that's going to create problems later. Why not fix it now?

Reply:
Ok, I'll make it into a feature.